tests/details/sparc.yaml

```diff
@@ -927,3 +927,26 @@ test_cases:
               cc: SPARC_CC_ICC_VS
               cc_field: SPARC_CC_FIELD_XCC
             # groups: [ HasV9 ] - Bug: llvm-project/#142388
+  -
+    input:
+      bytes: [ 0xa5, 0x61, 0xc0, 0x11 ]
+      arch: "sparc"
+      options: [ CS_OPT_DETAIL, CS_MODE_BIG_ENDIAN, CS_MODE_V9 ]
+      address: 0x0
+    expected:
+      insns:
+        -
+          asm_text: "movu	%fcc0, %l1, %l2"
+          details:
+            sparc:
+              operands:
```

Comment:
It is a conditional move instruction that reads fcc0 and conditionally copies l1 to l2. Why is fcc0 not added as the first operand?

Reply:
It is the CC field which is checked. This is the `cc_field` below.

Reply:
The cc fields are itself not an operand. But are a field in some register. For `fcc0`, `fcc1` tc it is the `fsr` register. for `xcc` and `icc` it is the `ccr` register.

This is why I added a `cc_field` member to the Sparc details. To have this property more separated from it.